.travis.yml

```
@@ -3,6 +3,7 @@ node_js:
 - 12
 - 14
 - 16
+- 20
```

As discussed offline, since travis is used as a way to validate the builds work for each version we should change this to:
```suggestion
- 18
- 20
- 22
- 23
```

That way we can know if something is going wrong before we move onto the next node version and the one before the one we use